target/linux/realtek/dts-5.15/rtl8393_netgear_gs750e.dts

```diff
@@ -190,7 +193,7 @@
 		SWITCH_PORT(7, 8, qsgmii)
 
 		SWITCH_PORT(8, 9, qsgmii)
-		SWITCH_PORT(9, 0, qsgmii)
+		SWITCH_PORT(9, 10, qsgmii)
```

Comment:
And also this?

Reply:
Same as the regmap above, with the extra bit that I'm near 100% sure the original was a typo.

Reply:
Can you split this out into a separate commit?